contracts/serverless-v2/Relay.sol

```diff
@@ -1052,7 +1087,22 @@ contract Relay is
         // TODO: do we need to check this bool success
         (bool success, ) = _jobOwner.call{value: callbackAmount}("");
 
-        emit JobSubscriptionTerminated(_jobSubsId, _jobOwner, usdcAmount, callbackAmount, success);
+        emit JobSubscriptionFundsWithdrawn(_jobSubsId, _jobOwner, usdcAmount, callbackAmount, success);
+    }
+
+    function _terminateJobSubscription(
```

Comment:
what if this function is not defined and external terminateJobSubscription method calls _updateJobSubsTerminationParams directly with block.timestamp + overalltimeout

Reply:
yeah can be done with some modifications in the `_updateJobSubsTerminationParams()` function

Reply:
also, it will only emit event `JobSubscriptionTerminationParamsUpdated()` , not the `JobSubscriptionTerminated()` event, right?

Reply:
Keeping the JobSubscriptionTerminated also would be preferable. Good for  clients.